src/main/java/com/wine/to/up/recommendation/service/domain/entity/Recommendation.java

```diff
@@ -1,16 +1,22 @@
 package com.wine.to.up.recommendation.service.domain.entity;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.*;
 
-@Getter
-@Setter
-@AllArgsConstructor
+import javax.persistence.*;
+
+@Data
```

Comment:
Not sure about `@Data` annotation. Hibernate can act unpredictably when it has overriden equals and hashCode